docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  openbio:
+    build: .
+    stdin_open: true
+    tty: true
```

Comment:
I see why you added this.  You have `run` in the docs so it should be moot, but I can see people running `up` out of habit.

PS - sorry, for the one off replies.  I should have done the proper review flow -- I only intended to leave a single comment 🙈

Reply:
np!

Reply:
She is trying to avoid 

```
[9:57:40][cayla@wopr]% cat docker-compose.yaml                                                                                                                                                                                                                                                                                                                        (cayla-k8s.addgene.org)
---

services:

  ubuntu:
    command: bash
    image: ubuntu:22.04
    # stdin_open: true
    # tty: true
    volumes:
      - ${OPENBIO_WORKDIR:-./workdir}:/home
[~/dev/tmp]
[9:57:43][cayla@wopr]% docker compose up                                                                                                                                                                                                                                                                                                                              (cayla-k8s.addgene.org)
[+] Running 1/0
 ✔ Container tmp-ubuntu-1  Created                                                                                                                                                                                                                                                                                                                                                       0.0s
Attaching to tmp-ubuntu-1
tmp-ubuntu-1 exited with code 0
[~/dev/tmp]
[9:57:45][cayla@wopr]% vim docker-compose.yaml                                                                                                                                                                                                                                                                                                                        (cayla-k8s.addgene.org)
[~/dev/tmp]
[9:57:51][cayla@wopr]% cat docker-compose.yaml                                                                                                                                                                                                                                                                                                                        (cayla-k8s.addgene.org)
---

services:

  ubuntu:
    command: bash
    image: ubuntu:22.04
    # stdin_open: true
    tty: true
    volumes:
      - ${OPENBIO_WORKDIR:-./workdir}:/home
[~/dev/tmp]
[9:57:54][cayla@wopr]% docker compose up                                                                                                                                                                                                                                                                                                                              (cayla-k8s.addgene.org)
[+] Running 1/0
 ✔ Container tmp-ubuntu-1  Recreated                                                                                                                                                                                                                                                                                                                                                     0.0s
Attaching to tmp-ubuntu-1
tmp-ubuntu-1  | root@9e3740d54bd2:/#


^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...
[+] Stopping 1/1
 ✔ Container tmp-ubuntu-1  Stopped                                                                                                                                                                                                                                                                                                                                                      10.2s
canceled
[~/dev/tmp]
[9:58:12][cayla@wopr][130] % cat docker-compose.yaml                                                                                                                                                                                                                                                                                                                        (cayla-k8s.addgene.org)
---

services:

  ubuntu:
    command: bash
    image: ubuntu:22.04
    stdin_open: true
    volumes:
      - ${OPENBIO_WORKDIR:-./workdir}:/home
[~/dev/tmp]
[9:58:51][cayla@wopr]% docker compose up                                                                                                                                                                                                                                                                                                                              (cayla-k8s.addgene.org)
[+] Running 1/0
 ✔ Container tmp-ubuntu-1  Recreated                                                                                                                                                                                                                                                                                                                                                     0.0s
Attaching to tmp-ubuntu-1


^CGracefully stopping... (press Ctrl+C again to force)
Aborting on container exit...
[+] Stopping 1/1
 ✔ Container tmp-ubuntu-1  Stopped                                                                                                                                                                                                                                                                                                                                                      10.1s
canceled
[~/dev/tmp]
[9:59:12][cayla@wopr][130] %    
```

Aka for when people try to `up` instead of `run`.